doc_source/update-cluster.md

```diff
@@ -180,24 +180,244 @@ The cluster update should finish in a few minutes\.
 
 ------
 
-1. Patch the `kube-proxy` daemonset to use the image that corresponds to your cluster's Region and current Kubernetes version \(in this example, `1.16.8`\)\.    
-[\[See the AWS documentation website for more details\]](http://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html)
-
-   First, retrieve your current `kube-proxy` image:
-
+Update `kube-proxy` on Worker nodes:
+- Check existing `kube-proxy` DaemonSet configuration on the cluster,
```

Comment:
What are we checking here ? Can you please re-phrase this sentence.

Reply:
kube-proxy DaemonSet configuration on clusters created before 1.12 version does not use ConfigMap to pull configuration. Clusters with kube-proxy without configMap should be updated to use configMap. 
I will brief this in the doc.